auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/controller/UserController.java

```diff
@@ -0,0 +1,84 @@
+package com.atlantbh.auctionappbackend.controller;
+
+import com.atlantbh.auctionappbackend.domain.User;
+import com.atlantbh.auctionappbackend.security.JwtConfig;
+import com.atlantbh.auctionappbackend.service.UserService;
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.*;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequestMapping(path = "${application.api.prefix}/user")
+public class UserController {
```

Comment:
I suggest creating `AuthController` which should be used only for those `auth` things.

Reply:
Sure.